roosevelt.js

```diff
@@ -176,7 +176,7 @@ module.exports = function (params) {
     }
 
     function bundleJs () {
-      require('./lib/jsBundler')(app, compileJs)
+      require('./lib/jsBundler').bundle(app, compileJs)
```

Comment:
Without this change, `npm run test:integration` still passes the first 8 or so tests for `jsBundler`. I'm not sure if a fresh run behaves that way, or if I had phantom files leftover from a previously successful run.